modules/microshift-oc-adm-by-example-content.adoc

```diff
@@ -3,116 +3,30 @@
 // Uses 'source,bash' for proper syntax highlighting for comments in examples
 
 :_content-type: REFERENCE
-[id="microshift-oc-cli-admin_{context}"]
+[id="openshift-cli-admin_{context}"]
```

Comment:
These IDs need to say microshift or it will break the repo. Our commands are different.